cmds/exp/tsort/multiset.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2012-2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import "fmt"
+
+func makeMultiset() multiset {
+	return multiset{m: map[string]int{}}
+}
+
+type multiset struct {
```

Comment:
I think you can simplify this to
```go
type multiset map[string]int
```

and then you can remove all of m.m

Reply:
See https://github.com/u-root/u-root/pull/2984#discussion_r1605815108